tgui/packages/tgui/interfaces/common/MalfAiModules.tsx

```diff
@@ -42,8 +42,8 @@ export function MalfAiModules(props) {
     <GenericUplink
       categories={categoriesList}
       items={items}
-      currency={`${processingTime} PT`}
-      handleBuy={(item) => act('buy', { name: item.name })}
+      currency={`${processingTime} ВО`}
+      handleBuy={(item) => act('купить', { name: item.name })}
```

Comment:
```suggestion
      handleBuy={(item) => act('buy', { name: item.name })}
```
Это функция под названием `buy`!